pedpy/plotting/plotting.py

```diff
@@ -54,6 +60,187 @@ def _plot_series(  # pylint: disable=too-many-arguments
     return axes
 
 
+def _plot_multiple_series(  # pylint: disable=too-many-arguments
+    axes: matplotlib.axes.Axes,
+    title: str,
+    x: pd.Series,
+    y_s: list[pd.Series],
+    colors: list[str],
+    labels: list[str],
+    line_width: float,
+    x_label: str,
+    y_label: str,
+) -> matplotlib.axes.Axes:
+    axes.set_title(title)
+    for i in range(len(y_s)):
```

Comment:
Sorry, thought yesterday it is an already fixed warning.

You can use:
```python
# using enumerate
for i,_ in enumerate(y_s):
    ...

# using zip (slightly better imo)
for y, color, label in zip(y_s, colors, labels):
    axes.plot(
        x, y, color=color, label=label, linewidth=line_width
    )
```

Reply:
done